databases/mysql57/Portfile

```diff
@@ -161,14 +160,7 @@ if {$subport eq $name} {
         }
     }
 
-    variant openssl description {Enable OpenSSL support} {
-        depends_lib-append      path:lib/libssl.dylib:openssl
-
-        patchfiles-delete       patch-cmake-install_layout.cmake-yassl.diff
-
-        configure.args-delete   -DWITH_SSL:STRING=bundled
-        configure.args-append   -DWITH_SSL:STRING=${prefix}
-    }
+    variant openssl description {Enable OpenSSL support} {}
```

Comment:
How should this variant be handled now that it effectively does what the port normally does? Is it safe to remove or is migrate current users away from this variant? I would guess it's a dummy variant rather than a legacy compatibility one that activates some other variant instead.

Reply:
A member has advised over IRC that the variant can be removed (thanks @neverpanic !)